### PR TITLE
Added sp for produced detail 470112

### PR DESCRIPTION
**Pull request by SalmanQayyumEviden**

### CODE REVIEW

**Review by hardikkumar-desai-external — CHANGES_REQUESTED** [review]

File: src/EPR.CommonDataService.Data/Scripts/Stored Procedures/get-producer-details-by-organisationid.sql [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        FROM 
+            rpd.companyDetails
+        WHERE 
+            organisation_id = @organisationId 
+            AND subsidiary_id IS NULL
+        GROUP BY 
+            organisation_id
+    )
+
+    SELECT
```
[the remark this answers is not in the document]
Comment (hardikkumar-desai-external) [reply src/EPR.CommonDataService.Data/Scripts/Stored Procedures/get-producer-details-by-organisationid.sql:25]: just to clearify in your select statement add as for each column name to match above `organisation_size as ProducerSize` etc

[the remark this answers is not in the document]
Comment (SalmanQayyumEviden) [reply src/EPR.CommonDataService.Data/Scripts/Stored Procedures/get-producer-details-by-organisationid.sql:23]: done

**Review by hardikkumar-desai-external — DISMISSED** [review]

**Review by SebastianJLTaylor — APPROVED** [review]
Only reviewing the SP script:

I can see the duplicates are fixed since the join to the latestfile CTE is now in place.
I can also see the new CTE for latest submission and join to this.
This means the count of subsidiaries doesn't get inflated which is better 
I can also see that the LEFT JOIN to latestSubmission CTE has been changed to an INNER JOIN which i think will also be more efficient.

**Review by hardikkumar-desai-external — APPROVED** [review]

**Review by JulianYoungEviden — APPROVED** [review]
Code looks good...

**Pull request merged**